### ISSUE TITLE
cocomanga.com Anti-Adblock

### ISSUE THREAD

**Issue by erosman** [uBlockOrigin/uAssets#11837]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

https://www.cocomanga.com/*/*/*.html
https://www.cocomanga.com/11457/1/309.html

### Describe the issue

> 系统核心代码加载失败，这可能是由于浏览器广告过滤功能误屏蔽了网站功能代码，请在浏览器设置中关闭广告过滤后再刷新页面！ 

Google translate:

> The core code of the system failed to load. This may be because the browser advertisement filtering function has mistakenly blocked the website function code. Please turn off the advertisement filtering in the browser settings and then refresh the page! 

### uBlock Origin version

1.41.6

### Browser name and version

Firefox Nightly 99.0a1

### Settings

default

✅ uBlock filters – Annoyances
✅ CHN: AdGuard Chinese (中文)

### Notes

Previously: #11140, #11322

**Comment by okiehsch** [comment]
Already fixed in `AdGuard Chinese`.
`cocomanga.com##+js(nostif, checkSiteNormalLoad)`
works fine on my end.

**Comment by uBlock-user** [comment]
Test on default settings/lists + all other extensins disabled.

**Comment by erosman** [comment]
> Test on default settings/lists + all other extensins disabled.

The settings are default. I have no other extension that would interfere. (All other extensions are my own)

Have you tested on Nightly?

**Comment by uBlock-user** [comment]
I'm on Nightly, can't reproduce. Same on Chromium too.

**Comment by mapx-** [comment]
you could try

`@@||cocomanga.com^$xhr,1p`

**Comment by erosman** [comment]
Is this how your looks too?

![image](https://user-images.githubusercontent.com/6458876/155147423-b458902e-9a64-49c4-8dd4-19963ddab0bf.png)

![image](https://user-images.githubusercontent.com/6458876/155147609-836eb4c3-30c6-4177-a492-547b851cc766.png)

**Comment by uBlock-user** [comment]
> I have no other extension that would interfere. (All other extensions are my own)

Do you mind disabling them ? This is to prove this is not a uBO issue.

**Comment by erosman** [comment]
Here is a copy of the log

<details><summary>Logger output</summary>

```
+1
@@||cocomanga.com/js/ad*/*.js
++
www.cocomanga.com
1
xhr
https://www.cocomanga.com/js/ad_/ad2.js
```
```
+1
www.cocomanga.com
3
image
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/favicon.png
```
```
+1
www.cocomanga.com
1
image
https://www.cocomanga.com/favicon.png
```
```
+1
www.cocomanga.com
3
image
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/favicon.png
```
```
+1
www.cocomanga.com
1
image
https://www.cocomanga.com/favicon.png
```
```
+1
||exdynsrv.com^
--
www.cocomanga.com
3
script
https://a.exdynsrv.com/fp-interstitial.js
```
```
+1
||exdynsrv.com^
--
www.cocomanga.com
3
script
https://a.exdynsrv.com/fp-interstitial.js
```
```
+1
*$script,3p,domain=cocomanga.com
--
www.cocomanga.com
3
script
https://couplestupidity.com/8c/0a/37/8c0a375d8544f78968ab9f8f99cb9fd7.js
```
```
+1
||exdynsrv.com^
--
www.cocomanga.com
3
script
https://a.exdynsrv.com/ad-provider.js
```
```
+1
||exdynsrv.com^
--
www.cocomanga.com
3
script
https://a.exdynsrv.com/js.php?t=17&idzone=3945126
```
```
+1
||exdynsrv.com^
--
www.cocomanga.com
3
script
https://a.exdynsrv.com/js.php?t=17&idzone=3945124
```
```
+1
||exdynsrv.com^
--
www.cocomanga.com
3
script
https://a.exdynsrv.com/nativeads-v2.js
```
```
+1
/cdn-cgi/bm/cv/*
--
www.cocomanga.com
1
script
https://www.cocomanga.com/cdn-cgi/bm/cv/669835187/api.js
```
```
+1
||zz.bdstatic.com^
--
www.cocomanga.com
3
script
https://zz.bdstatic.com/linksubmit/push.js
```
```
+1
||cnzz.com^$3p
--
www.cocomanga.com
3
script
https://s4.cnzz.com/z_stat.php?id=1280242613&web_id=1280242613
```
```
+0
##+js(addEventListener-defuser, load, popMagic)
www.cocomanga.com
dom
https://www.cocomanga.com/22449/1/65.html
```
```
+0
##+js(set-constant, atob, undefined)
www.cocomanga.com
dom
https://www.cocomanga.com/22449/1/65.html
```
```
+0
##+js(no-setTimeout-if, checkSiteNormalLoad)
www.cocomanga.com
dom
https://www.cocomanga.com/22449/1/65.html
```
```
+0
##+js(set-constant, __jsadsuccess, true)
www.cocomanga.com
dom
https://www.cocomanga.com/22449/1/65.html
```
```
+0
##+js(set-constant, NativeAd, noopFunc)
www.cocomanga.com
dom
https://www.cocomanga.com/22449/1/65.html
```
```
+0
##+js(addEventListener-defuser, contextmenu)
www.cocomanga.com
dom
https://www.cocomanga.com/22449/1/65.html
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/template/vfed/asset/js/global.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/template/vfed/asset/js/global.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/template/vfed/asset/js/jquery.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/template/vfed/asset/js/jquery.js
```
```
+0
/cdn-cgi/bm/cv/*
--
www.cocomanga.com
1
script
https://www.cocomanga.com/cdn-cgi/bm/cv/669835187/api.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/js/dynamicjs.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/js/dynamicjs.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/js/custom.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/js/custom.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/js/jquery.cookie.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/js/jquery.cookie.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/js/jquery-3.3.1.min.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/js/jquery-3.3.1.min.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/js/manga.read.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/js/manga.read.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/js/common.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/js/common.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/js/style.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/js/style.js
```
```
+0
@@||cdn.cloudflare.net^$script,domain=cocomanga.com|cocomanga.com
++
www.cocomanga.com
3
script
https://cdn.cocomanga.com.cdn.cloudflare.net/
aliasURL=https://www.cocomanga.com/js/l.js
```
```
+0
www.cocomanga.com
1
script
https://www.cocomanga.com/js/l.js
```
```
+0
www.cocomanga.com
1
doc
https://www.cocomanga.com/22449/1/65.html
```
```
+0
https://www.cocomanga.com/22449/1/65.html
```
</details>

**Comment by uBlock-user** [comment]
Something to try --

- Create a new profile and install uBO only there, update the lists once you install uBO and activate AdGuard Chinese if it's not activated.

**Comment by erosman** [comment]
Does the log say anything?

**Comment by erosman** [comment]
I got it working. Since yesterday, Nightly has some issues with its caching which might be related.
Just for info ....

- I disabled both uBO & uMatrix and restated Firefox
- Went to the above domain and there was still the issue present
- Hard Refreshed the page
- Pages started to work (with ads)
- Enabled uBO & uMatrix and it is fine now

It is possible that the caching issues has affected uBO and/or the page.
